app/lib/constants/colors.ts

```diff
@@ -157,7 +157,7 @@ const dark = {
 	buttonBackgroundPrimaryPress: '#245399',
 	buttonBackgroundPrimaryDisabled: '#1D3963',
 
-	buttonBackgroundSecondaryDefault: '#2F343D',
+	buttonBackgroundSecondaryDefault: '#353B45',
```

Comment:
We should find a proper color for black as well.

Reply:
![image](https://github.com/user-attachments/assets/65be0bd0-6f67-44d7-9f9b-01cb3929e912)